src/Exchange.jl

```diff
@@ -1,7 +1,7 @@
 """
     module Exchange
 
-Here we include methods for calculating the metropolis condition and other exchange criteria required for Monte Carlo steps. This further declutters the MCRun module and allows us to split the cycle
+Here we include methods for calculating the metropolis condition and other exchange criteria required for Monte Carlo steps. This further declutters the MCRun module and allows us to split the cycle. Includes update_max_stepsize which controls the frequency of
 """
```

Comment:
incomplete documentation: the frequency of what?

Reply:
What is the logic behind including update_max_stepsize here? I think it should be in Moves.jl?